CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.6.1
```

Comment:
Wondering if, given that the way we "package" our release manifests has changed, all releases should be a new MINOR for semver predictability/correctness.